charts/kube-vip-cloud-provider/templates/deployment.yaml

```diff
@@ -16,14 +16,31 @@ spec:
       containers:
         - command:
             - /kube-vip-cloud-provider
-            - --leader-elect-resource-name=kube-vip-cloud-controller
+            - --leader-elect-resource-name={{ include "kube-vip-cloud-provider.name" . }}
+            {{- if kindIs "map" .Values.extraArgs }}
+            {{- range $key, $value := .Values.extraArgs }}
+            {{- if not (kindIs "invalid" $value) }}
+            - --{{ $key }}={{ tpl ($value | toString) $ }}
```

Comment:
`tpl` should only be needed if the value is in itself meant to be dereferenced as a value, like "{{ .Values.something }}", right?  Shouldn't $value just be a string?